pandas/tests/io/formats/style/test_style.py

```diff
@@ -1548,3 +1548,9 @@ def test_col_trimming_hide_columns():
         assert ctx["head"][0][c + 2]["is_visible"] == vals[1]
 
     assert len(ctx["body"][0]) == 6  # index + 2 hidden + 2 visible + trimming col
+
+
+def test_no_empty_apply(mi_styler):
+    # 45313
+    mi_styler.apply(lambda s: ["a:v;"] * 2, subset=[False, False])
+    mi_styler._compute()
```

Comment:
need a comparison here

Reply:
This tests it just computes without raising (fails on master since it raises). We need a comparison to a correctly styled frame?

Reply:
ok its fine then